pkg/config/validation/validation.go

```diff
@@ -1734,6 +1734,16 @@ var ValidatePeerAuthentication = registerValidateFunc("ValidatePeerAuthenticatio
 				fmt.Errorf("peer authentication with empty workload selector must be named %q", constants.DefaultAuthenticationPolicyName))
 		}
 
+		if emptySelector && len(in.PortLevelMtls) != 0 {
+			errs = appendErrors(errs,
+				fmt.Errorf("default peer authentication cannot have port level mTLS"))
```

Comment:
`default` here does it mean mesh/namespace level authn? can we reword to `mesh/namespace peer authentication cannot have...` to be more specific?

"default" is often overloaded i feel

Reply:
peer authn.. without selector cannot have ...

Reply:
done